### PR TITLE
added LOGIN credentials to readme file

### PR DESCRIPTION
**Pull request by artofmayhem**
Administrative FIX - Added user login credentials to the readme.md file for future reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,6 +185,15 @@ Back End
 - npm start
 ```
 
+
+
+## LOGIN Credentials
+User: llama001@maildrop.cc
```
Comment (just-kato) [inline README.md:191]: BIG BIG help. I was looking for this info everywhere

**Review by just-kato — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MichaelMcFaddenjr) [reply README.md:191]: Thank you for adding this. I have had to search through slack!

**Review by MichaelMcFaddenjr — APPROVED** [review]

**Review by Yonathan-Admasu728 — APPROVED** [review]
this is huge help. Thank you!

File: README.md (hunk 1 above)
Comment (rhiannonstanford) [reply README.md:191]: Very nice to have this info in a designated place.  Thank you!

**Review by rhiannonstanford — APPROVED** [review]

**Pull request merged**